src/components/Layout.jsx

```diff
@@ -1,27 +1,26 @@
 import React from 'react';
+import { Container } from '@mui/material';
+import PropTypes from 'prop-types';
 import Header from './Header';
 import Footer from './Footer';
-import { Box } from '@mui/material';
-import PropTypes from 'prop-types';
+
+import { commonStyles } from '../constants';
 
 const Layout = ({ children }) => {
   return (
-    <Box
+    <Container
+      display='flex'
+      flexDirection='column'
+      maxWidth='xl'
       sx={{
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        justifyContent: 'center',
-        minHeight: '100vh',
-        backgroundColor: '#282c34',
-        color: 'white',
-        fontSize: 'calc(10px + 2vmin)',
+        backgroundColor: commonStyles.backgroundColor,
```

Comment:
[req] [body color](https://github.com/Umka-dev/Rick-and-Morty-characters/pull/9/files#diff-85da366246340e911d7a0eb9153e64137a1b31f0686c74d9aef9ae4f032cb09eR8) does the same job and `backgroundColor` here could be removed
```suggestion
        backgroundColor: commonStyles.backgroundColor,
```